### ISSUE TITLE
quickstart.py --no-batchfile isn't honored

### ISSUE THREAD

**Issue by ktreese** [sphinx-doc/sphinx#5557]
Subject: `--no-batchfile` still prompts user whether they'd like to "Create Windows command file?"

### Problem
`--no-batchfile` still prompts user whether they'd like to "Create Windows command file?"

#### Procedure to reproduce the problem
```
~ # sphinx-quickstart --sep --dot _ --project kristianreese.com --author 'Kris Reese' -v 1.0 --release 0 -l en --suffix .rst --master
 index --ext-autodoc --ext-todo --ext-ifconfig --makefile --no-batchfile
Welcome to the Sphinx 1.8.1 quickstart utility.

Please enter values for the following settings (just press Enter to
accept a default value, if one is given in brackets).

Selected root path: .
> Create Windows command file? (y/n) [y]: 
```

#### Error logs / results
```
User should not be  prompted
```
### Environment info
- OS: Linux
- Python version: 3.7.0
- Sphinx version:1.8.1

**Comment by ktreese** [comment]
nor is --no-makefile

```
/test # sphinx-quickstart --sep --dot _ --project mywebsite.com --author 'Kris Reese' -v 1.0 --release 0 -l en --suffix .rst --master index --ext-autodoc --ext-todo --ext-ifconfig --no-makefile --no-batchfile
Welcome to the Sphinx 1.8.1 quickstart utility.

Please enter values for the following settings (just press Enter to
accept a default value, if one is given in brackets).

Selected root path: .

A Makefile and a Windows command file can be generated for you so that you
only have to run e.g. `make html' instead of invoking sphinx-build
directly.
> Create Makefile? (y/n) [y]: n
> Create Windows command file? (y/n) [y]: n
```

**Comment by tk0miya** [comment]
Fixed by #5581.
Thank you for reporting.
